src/app/util/attribute-table.c

```diff
@@ -657,3 +658,54 @@ EmberAfStatus emAfReadAttribute(uint8_t endpoint, EmberAfClusterId cluster, Embe
 
     return status;
 }
+
+EmberAfAttributeWritePermission emberAfAllowNetworkWriteAttributeCallback(uint8_t endpoint, EmberAfClusterId clusterId,
+                                                                          EmberAfAttributeId attributeId, uint8_t mask,
+                                                                          uint16_t manufacturerCode, uint8_t * value, uint8_t type)
+{
+    return EMBER_ZCL_ATTRIBUTE_WRITE_PERMISSION_ALLOW_WRITE_NORMAL; // Default
+}
+
+EmberAfStatus emberAfPreAttributeChangeCallback(uint8_t endpoint, EmberAfClusterId clusterId, EmberAfAttributeId attributeId,
```

Comment:
This makes this callback non-overridable by apps, right?  Is that intended?  Similar for the other callbacks added here.

Reply:
All callbacks that were moved here were already used by all examples as is and couldn't be generated with zap. Should the need arise to overwrite them by application then we shall find another solution.